containerd.service

```diff
@@ -18,7 +18,6 @@ Documentation=https://containerd.io
 After=network.target local-fs.target
 
 [Service]
-ExecStartPre=-/sbin/modprobe overlay
```

Comment:
Please do not remove this.

Reply:
I'm sorry, but that's the reason of this pull request; to remove exactly this line, because in my opinion it doesn't belong there.

Reply:
> Also for this module file, I'm wondering if we could add it to contrib. We usually put files for working with external components there to reduce the number files in root.

This would be worth considering. However, in my eyes this leads to the question whether the service file `container.service` should not also be moved to the folder `/contrib`?

I would then extend the pull request to that effect.

Reply:
That breaks existing installations

Reply:
There is a central place where kernel modules to be loaded are stored for a reason. See pull request description.

Reply:
> That breaks existing installations

This is not the case, or should not be the case, if the package maintainer includes the appropriate module file in the package.

Reply:
Then we have to postpone this to v2.0 milestone.